renku/service/jobs/cleanup.py

```diff
@@ -16,37 +16,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Cleanup jobs."""
-import json
 import os
 import shutil
 import time
 
 from renku.service.cache import FileManagementCache, ProjectManagementCache
-from renku.service.utils import make_file_path, make_project_path
 
 
 def cache_files_cleanup():
     """Cache files a cleanup job."""
     ttl = int(os.getenv('RENKU_SVC_CLEANUP_TTL_FILES', 1800))
     cache = FileManagementCache()
 
-    for file_key in cache.all_files_iter():
-        try:
-            user, _ = file_key.decode('utf-8').split('_')
-        except ValueError:
-            continue
-
-        user = {'user_id': user}
-        for file_meta in cache.hash_table(file_key).values():
-            data = json.loads(file_meta)
-
+    for user, files in cache.user_files():
+        for file in files:
             # If record does not contain timestamp,
             # it means its an old record, and we should mark it for deletion.
-            created_at = data.get('timestamp', (time.time() - ttl) * 1e+3)
+            created_at = file.timestamp
```

Comment:
`Job` has `created_at` and `updated_at` but `File` uses `timestamp` instead of `created_at`, we should keep naming consistent. `Project` also uses `timestamp`.